tests/util/translator/parse/common/parse-base-schema.spec.tsx

```diff
@@ -27,7 +27,12 @@ describe("ParseBaseSchema", () => {
             uiType: EElementType.TEXT,
         })[elementId] as TElementSchema;
 
-        const parsedSchema = parseBaseSchema(mockSchema, elementId, {});
+        const parsedSchema = parseBaseSchema(
+            mockSchema,
+            elementId,
+            {},
+            undefined
```

Comment:
once the `defaultValue` argument is made optional, you can skip specifying the last argument